config/navigation.json

```diff
@@ -35,7 +35,7 @@
       "group": "foundation",
       "mode": "video",
       "description": "Generate videos from text or image prompts using Veo.",
-      "video_url": "https://deepmind.google/api/blob/website/media/veo__cover_s0RKXWX.mp4"
+      "video_url": "https://storage.googleapis.com/gdm-deepmind-com-prod-public/media/media/veo__page-cover-1_hcTy7l3.mp4"
```

Comment:
Apologies, keep this tile video as original veo video:

https://storage.googleapis.com/gdm-deepmind-com-prod-public/media/media/veo__cover_s0RKXWX.mp4